### ISSUE TITLE
add specification for merged /usr filesystem layout

### ISSUE THREAD

**Issue by craftyguy** [uapi-group/specifications#120]
`man 7 file-hierarchy` from systemd details how the filesystem should be laid out for merged /usr, but IMHO this should be a specification under UAPI so that distros can discover, implement, and reference it more easily.

**Comment by pabloyoyoista** [comment]
Coincidentally, we have started the process to update the FHS to include the /usr merge: https://gitlab.freedesktop.org/freedesktop/freedesktop/-/issues/1695

I had a discussion some weeks ago (among others, with @Conan-Kudo) regarding this. We had the understanding that the FHS has a wider scope, and also applies to more things than Linux, and therefore would be best fitting for it to live under the FDO than the UAPI umbrella. If folks here are willing to consider other Unixes and endorse the current scope of the FHS according to the Linux Foundation, then maybe doing it here would make sense?

**Comment by Conan-Kudo** [comment]
> The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, and run modern Linux operating systems.

This is probably not the appropriate place for a cross OS umbrella.

**Comment by poettering** [comment]
Yeah, we don't really bother with non-Linux stuff on uapi-group.org. 

I'd say whoever spearheads the work, gets the last call on this. From our perspective we have the text in https://www.freedesktop.org/software/systemd/man/latest/file-hierarchy.html which documents our (as in systemd's) perspective on things.

**Comment by bluca** [comment]
I don't think we should duplicate the entire file-hierarchy manpage or try to take over the dead FHS. However, I don't think that is the request here. We mention hermetic-usr a lot, and usr-merge is fundamental for that, so I think it makes a lot of sense to have a page dedicated to the hermetic-usr and related required layout on uapi-group.org. This will provide clarity, ensure it's clear it's not a systemd-specific thing but everyone can benefit from it, and help the pmOS folks as well.

**Comment by pabloyoyoista** [comment]
It seems like recent discussions (e.g: https://lwn.net/ml/all/175464403192.753219.17972987109162454274@mailman01.rdu3.fedoraproject.org/) and follow-up work in here means this has actually been implemented and can be closed?
